core/src/test/java/org/infinispan/util/PersistenceMockUtil.java

```diff
@@ -41,6 +47,51 @@
  */
 public class PersistenceMockUtil {
 
+   public static class Builder {
```

Comment:
`ContextBuilder` as `PersistenceMockUtil` is not exclusive to `InvocationContext`?

Reply:
Sorry, I am not sure what you mean here. Are you proposing I move the Builder class somewhere else?

Reply:
Sorry, my message isn't clear in hindsight. I meant shall we rename to `class ContextBuilder` as the `PersistenceMockUtil` class is not exclusively used for creating/modifying `InvocationContext` instances. The name `Builder` is fine as an inner class that creates an instance of the parent class, e.g. `DataFormat.Builder`, however that is not the case with  `PersistenceMockUtil.Builder`.

Reply:
Sounds good. I guess because the class only creates an InvocationContext currently I hadn't thought about the name, but I agree.